README.md

```diff
@@ -4,30 +4,60 @@
 [![Platform](https://img.shields.io/cocoapods/p/WMobileKit.svg?style=flat)](http://cocoapods.org/pods/WMobileKit)
 
-WMobileKit is a Swift library containing various custom UI components to provide functionality outside of the default libraries.
+WMobileKit is a Swift library containing various custom UI components to provide functionality outside of the default libraries. Each component is customizable and featured in a sample app to demonstrate its functinality!
+
+### Index
+* [Controls](#controls)
+* [Information Views](#information-views)
+* [Modal Components](#modal-components)
+* [Navigation](#navigation)
+* [Text Inputs](#text-inputs)
+* [Miscellaneous/Utilities](#miscellaneousutilities)
 
 ## Features
 
-- Customizable
-- Example App
-- Components
- - [x] WActionSheet
- - [x] WAutoCompleteTextView
- - [x] WBadge
- - [x] WBanner
- - [x] WLabel
- - [x] WLoadingModal
- - [x] WRadioButton
- - [x] WSideMenuVC
- - [x] WSizeVC
- - [x] WSpinner
- - [x] WSwitch
- - [x] WTextField
- - [x] WTextView
- - [x] WTheme
- - [x] WToast
- - [x] WUserLogoView
- - [x] WUtils
+### Controls
+Component | Demo
+--- | ---
+<b>WRadio</b><br> Web-like radio button that can be tied to a group and automatically select/deselect as expected from a normal radio button. The size and color of the outer and inner circle can be changed as well as the highlight. | <img src="Gifs/WSwitchWRadio.gif">
+<b>WSwitch</b><br> Custom switch control emulating Apple's custom switches. Adds the ability to change sizes, colors, and sliding behaviors. | <img src="Gifs/WSwitchWRadio.gif">
+
+### Information Views
+Component | Demo
+--- | ---
+<b>WBadge</b><br> Custom number badge that provides the ability to change color, spacing, shape, location, and how the number displays/expands. | <img src="Gifs/WBadge.gif">
+<b>WSpinner</b><br> Custom progress spinner allowing for custom colors, the addition of images,  | <img src="Gifs/WSpinner.gif">
+<b>WUserLogoView</b><br> An image used to represent a user. Can use either a name which is hashed into a color or and image that is cropped to fit the view. | <img src="Gifs/WUserLogoView.gif">
+
+### Modal Components
+Component | Demo
+--- | ---
+<b>WActionSheet</b><br> Custom action sheets providing the ability to include a picker view, automatically scale to content, scroll, include a cancel button, and tap to dismiss. | <img src="Gifs/WActionSheet.gif">
+<b>WBanner</b><br>  Static banners that display information over content from the top or bottom. Can be automatically dismissed or on tap. | <img src="Gifs/WBanner.gif">
+<b>WLoadingModal</b><br> Displays a loading view over content. Can dim the background view and will disappear after a set period of time. | <img src="Gifs/WLoadingModal.gif">
+<b>WToast</b><br> Custom toast that displays from the top/bottom or sides and can dismiss on top or a timer. The color, text, and transparency can be customized. | <img src="Gifs/WToast.gif">
```

Comment:
` and can dismiss on top or a timer` sounds a bit funny. Not sure the best option to change to. Something along the lines of `can be customized to dismiss on tap or via a timer` maybe?